src/components/ExecuteButton.scss

```diff
@@ -0,0 +1,7 @@
+.ins-c-remediations-connection-status {
+  margin-right: 5px;
+}
+
+.ins-c-remediations-connection-status-error {
+  color: #E10000;
```

Comment:
Use `var(--pf-chart-global--danger--Color--100)` instead of direct hex color, also align this to one line.